src/jpgimage.cpp

```diff
@@ -159,6 +187,15 @@ void JpegBase::readMetadata() {
       std::copy(sizebuf.begin(), sizebuf.end(), buf.begin());
     }
 
+    auto itSofMarker = std::find_if(std::begin(jpegProcessMarkerTags), std::end(jpegProcessMarkerTags),
```

Comment:
oh operator== is already implemented. Exiv2::find can be used.

Reply:
Changed to `Exiv2::find`